rules/linux/process_creation/proc_creation_lnx_password_group_discovery.yml

```diff
@@ -0,0 +1,28 @@
+title: Potential Linux Groups Discovery
+id: 6d506805-389e-4370-8270-da7fd947a0da
+status: experimental
+description: This rule detects adversaries attempting to discover group and permission settings.
+    It looks for process execution related to `groups`, `getent`, or `netstat` commands which could indicate an attempt to gather information on available user accounts and groups, including elevated permissions.
+references:
+    - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1069.001/T1069.001.md
```

Comment:
Please use a permalink

And please add more references to the utilities being used.